src/ucp/wireup/wireup_cm.c

```diff
@@ -759,16 +759,21 @@ static void ucp_cm_disconnect_cb(uct_ep_h uct_cm_ep, void *arg)
     ucp_ep_h ucp_ep            = arg;
     uct_worker_cb_id_t prog_id = UCS_CALLBACKQ_ID_NULL;
     ucp_worker_h worker        = ucp_ep->worker;
-    uct_ep_h uct_ep;
 
     ucp_ep->flags |= UCP_EP_FLAG_DISCONNECT_CB_CALLED;
```

Comment:
BTW, unrelated but should not we do async block here?

Reply:
we should - it is done in #6312